apps/labs/posts/sympy-documentation.md

```diff
@@ -0,0 +1,351 @@
+docstrings, but the SymPy documentation has historically been lacking in
+long-form narrative documentation to supplement these reference docs.
+
+In this post, I will go over some of the key documentation improvements that
+were made over the course of the 2-year grant period. Note that the
+documentation improvements were only one part of the CZI grant to improve
+SymPy. Other SymPy developers were funded to [improve the performance of
+SymPy](https://oscarbenjamin.github.io/blog/czi/index.html), and to [improve
+its code generation
+capabilities](https://mechmotum.github.io/blog/czi-sympy-wrapup.html).
+
+# Documentation Survey
+
+To start the project, from November 29, 2021 to January 5, 2022 we ran a short
+survey on the SymPy community, to get a feel for SymPy's documentation needs.
+We had three takeaways.
+
+One - the main SymPy documentation site (https://docs.sympy.org) is overwhelmingly
+the most popular resource that people use to get help with SymPy. This is
+true across all levels of experience, compared to other resources like the
+SymPy website, StackOverflow, and community sites (note: this survey was
+given in 2021, before the popularity of LLM tools like ChatGPT, so this was
+not included as an option for respondents).
+Consequently, we decided that it would be most impactful to spend efforts on
+improving the documentation site over those other resources.
+
+![Plot showing results of SymPy documentation question "which of the
+  following resources do you use when getting help with SymPy (select all that
+  apply)?" The choice "SymPy Docs Website" has the most responses at
+  91%.](/posts/sympy-documentation/sympy-survey-results.svg)
+
+Two - survey respondents identified many deficiencies in the SymPy docs which
+made it clear that certain improvements needed to be made to the overall
+layout and organization of SymPy documentation site. In particular, we
+identified 4 major improvements that could be made:
+
+- Better top-level organization.
+- A better Sphinx theme that provides better sidebar navigation.
+- The docs have many large pages which would benefit from being split into smaller pages.
+- There were several issues with the SymPy Live extension. This was a
+  Sphinx extension that allowed users to execute the example code blocks in
+  the SymPy documentation directly in their browser.
+
+Of these, all except the third, splitting large pages, were done as part of the
+CZI grant work. Splitting large pages hasn't been done yet due to technical
+difficulties with the Sphinx autodoc extension, as well as due to the fact that
+the large pages are now much easier to navigate with the new Sphinx theme.
+
+The SymPy Live extension in the documentation was removed, as it was
+considered too much of a maintenance burden for the SymPy community to
+maintain. There is a new [SymPy Live](https://live.sympy.org/) shell that
+runs on [JupyterLite](https://jupyter.org/try-jupyter/lab/index.html)—that
```

Comment:
It's supposed to be an em dash. Some markdown processors recognize a double hyphen `--` as an em dash. Our site for some reason does not. Different writing style guides differ on whether to put spaces around the em dash. The AP guide says to surround the em dash with spaces, plus I think the em dash here looks better with spaces, so here's my suggestion:

```suggestion
runs on [JupyterLite](https://jupyter.org/try-jupyter/lab/index.html) &mdash; that
```